lib/hcloud/volume_resource.rb

```diff
@@ -12,6 +12,12 @@ def [](arg)
     end
 
     def create(size:, name:, automount: nil, format: nil, location: nil, server: nil, labels: {})
+      raise Hcloud::Error::InvalidInput, 'no name given' if name.blank?
```

Comment:
In network_resource you check for `raise Hcloud::Error::InvalidInput, 'no name given' if name.blank?` which is the "better check I think because it also checks for empty strings. So for "names" all over the place I would consistently use the blank? check (or always use the nil?)